thrift/contentatom.thrift

```diff
@@ -0,0 +1,5 @@
+include "atoms/tenfour.thrift"
+
+union ContentAtom {
+  1: tenfour.Content tenfour
```

Comment:
The only reason I used tenfour was because we are going to contain polls and so one, not just quizzes. If I call it a quiz then I am worried it will seem strange if it actually contains a poll.